nixos/modules/misc/ids.nix

```diff
@@ -340,6 +340,7 @@
       cockroachdb = 313;
       zoneminder = 314;
       paperless = 315;
+      btpd = 316;
```

Comment:
https://github.com/NixOS/nixpkgs/issues/60732#issuecomment-488829636

Reply:
I'm not exactly sure what I'm supposed to do here. Should I not set a uid at all, set it to null, or something else?

Reply:
Remove this line and remove the line in your module that assigns the uid/gid, which means it'll be set to `null`, the default value.